src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/PlaceRecognizer.scala

```diff
@@ -14,9 +14,11 @@ class PlaceRecognizer(
   @volatile private lazy val entityRecognizer = createEntityRecognizer()
 
   def extractPlacesAndOccurrance(text: String): Seq[(String, Int)] = {
-    entityRecognizer.extractEntities(text)
-      .filter(OpeNER.entityIsPlace)
-      .map(place => (place.getStr, place.getSpans.size()))
+    // See: https://github.com/opener-project/kaf/wiki/KAF-structure-overview
+    entityRecognizer.extractTerms(TextNormalizer(text, language.getOrElse("")))
+      .filter(t=>Set("N", "R").contains(t.getPos))
+      .groupBy(_.getStr)
+      .map(place => (place._1, place._2.size)).toSeq
```

Comment:
This is the other main change here; The `extractEntities` call was often returning an empty list when there were clearly identified terms marked as nouns in the OpeNER results.